.eslintrc.cjs

```diff
@@ -37,9 +37,11 @@ module.exports = {
 				"operator-assignment": "error",
 
 				// These on-by-default rules don't work well for this repo and we like them off.
+				"@typescript-eslint/padding-line-between-statements": "off",
```

Comment:
Curiosity: is there intent behind disabling this here, rather than removing its enable later in the file?

Reply:
Sheepish Admission: only because I didn't see it. Fixed!